docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,94 @@
+# Release 372 (23 Feb 2022)
+
+## General
+
+* Remove support for reserved memory pool. Configuration property
+  `experimental.reserved-pool-disabled` can no longer be used. ({issue}`6677`)
+* Ensure memory is released completely after query completion. ({issue}` 11030`)
+
+## Base-JDBC connector library
+
+* Fix spurious query failures when metadata cache is not enabled, and data
+  access depends on the session state. ({issue}`11068`)
+
+## Accumulo connector
+
+* Fix incorrect date results when querying `date` type columns. ({issue}`11055`)
+
+## Cassandra connector
+
+* Fix incorrect results when filtering partition keys without projections.
+  ({issue}`11001`)
+
+## ClickHouse connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Druid connector
+
+* Fix spurious query failures when metadata cache is not enabled, and extra
+  credentials with `user-credential-name` or `password-credential-name` are used
+  to access data. ({issue}`11068`)
+
+## Hive connector
+
+* Stop logging spurious failures when [storage caching](/connector/hive-caching)
+  is used. ({issue}`11101`)
+* Fix incorrect query results after writes to a table with caching directory
+  lister enabled with the `hive.file-status-cache-tables` configuration
+  property. ({issue}`10621`)
+* Fix potential query failures for queries writing data to tables backed by S3.
+  ({issue}`11089`)
+
+## Iceberg connector
+
+* Fix potential query failures for queries writing data to tables backed by S3.
+  ({issue}`11089`)
+
+## MongoDB connector
+
+* Support guessing MongoDB `bindata` type as Trino `varbinary` type.
```

Comment:
@ebyhr @hashhar  .. "guessing"? Either it maps or it doesnt right? If it truly guesses.. does that mean it maps to varbinary sometimes and not some other times? That would be really weird..

Reply:
It really is guessing. If an explicit schema is not provided to MongoDB connector it tries to guess column types based on the value that was read.

Maybe reword as

```suggestion
* Infer MongoDB `bindata` type as Trino `varbinary` type if explicit schema does not exist.
```

WDYT @ebyhr ?

Reply:
👍